Schedule_app/views.py

```diff
@@ -40,22 +40,29 @@ def bookticket(response):
         form = TicketBookForm()
         context = {'form': form}
         return render(response, 'bookticket.html', context)
-    # submitted = False
-    # if response.method == "POST":
-    #     form = TicketBookForm(response.POST)
-    #     if form.is_valid():
-    #         form.save()
-    #         return HttpResponseRedirect('/book_ticket?submitted=True')
-    # else:
-    #     form = TicketBookForm
-    #     if 'submitted' in response.GET:
-    #         submitted = True
-    # return render(response, 'bookticket.html', {"form": form, 'submitted': submitted, 'logedin': True})
     else:
         return render(response, 'bookticket.html', {'logedin': False})
 
 
 def cinema(response):
     form = TicketBookForm(response.GET)
-    print(form['cinema'])
+    print(response.GET)
```

Comment:
It's ok to use `print`s for debugging. But once you finish debugging and prepare your code for a commit, remove any kind of debugging method from the code. Lucky for us this doesn't interrupt the server, but you can never know.